configure

```diff
@@ -1352,6 +1352,10 @@ if (exists $opts{'gtest'} || $tests) {
         "default install dir '$sys_dir', please pass a correct version of Google Test to --gtest\nStopped";
     }
   }
+  # Check for existence of googletest shared / dynamic libraries
+  if (-d File::Spec->catdir($opts{'gtest'}, 'bin')) {
+    push_path(\%targetEnv, $opts{'gtest'} . $slash . 'bin');
```

Comment:
Should this be for Windows only?  Does the equivalent for `LD_LIBRARY_PATH` need to be added for Linux?

Or, if that gets too complicated... could we just go with archive/static libs for gtest?

Reply:
It already defaults to static, and if we build it as part of configure it's using the default / static. So this is really for when people pass in --gtest on the command line or if, for some reason, tests/googletest already contains the non-static variety. I just checked, and for Linux it looks like the .so's go into "lib" not "bin" which I guess is what I would have expected. I suppose, in that case, I need to add a special check for the .so file name to support shared libraries on Linux?

Reply:
It should probably be similar to Qt5 with something like `  push_libpath(\%targetEnv, $qt_path);` but with `$qt_path` replaced by the relevant gtest directory.  `push_libpath()` knows about the `PATH` vs. `LD_LIBRARY_PATH` distinction (plus the one that macOS uses).